### DISCUSSION TITLE
Sound problem

### DISCUSSION THREAD

**Discussion by ran09dom** [hydrogen-music/hydrogen#1228] [Hydrogen on Linux]
i am working with ubuntu 20.10 and Hydrogen 1.01and have the following problem: I open Hydrogen with a h2song file that plays without problems. However, when I play a video file on You Tube with Firefox, after this Hydrogen plays the h2song file only without sound. In the settings, the AlsaAudioDriver (automatically selected) is displayed under Audio system; under Midi system Alsa as driver.
ran09dom

**Comment by theGreatWhiteShark** [comment]
Hey @ran09dom ,

An application can get direct access to a sound card via ALSA. This approach will AFAIK not allow duplexing (playing more than one audio source in parallel by summing/mixing them). Maybe `Firefox` is holding on to the sound card. Could you shut it down (in a terminal using `killall firefox`) and trying again.

In general I would recommend you select the **PulseAudio** driver in the Audio tab of the Preferences dialog. It is advanced enough to handle various audio sources.

The MIDI settings are not a problem here.

**Reply by ran09dom** [reply]
Hey theGreatWhiteShark,
thank you very much for the explanations.I have set the Settings/Audio System under Audio Device PortAudio-Driver. Then the problem did not occur. I do not know if the setting is correct, it works.
Ran09dom

**Reply by theGreatWhiteShark** [reply]
> I do not know if the setting is correct, it works.

As long as it works it is correct :wink:
